options/posix/generic/pthread-stubs.cpp

```diff
@@ -47,7 +47,7 @@ struct ScopeTrace {
 #define SCOPE_TRACE() ScopeTrace(__FILE__, __LINE__, __FUNCTION__)
 
 static constexpr unsigned int mutexRecursive = 1;
-static constexpr unsigned int mutexErrorCheck = 2;
+[[maybe_unused]] static constexpr unsigned int mutexErrorCheck = 2;
```

Comment:
How come this is seen as unused?

Reply:
Oh this is similar to `mutex_excl_bit`, but the other way around. It's also defined and used in `threads.cpp`, and IG should be removed from here.